internal/service/advancedcluster/resource_advanced_cluster.go

```diff
@@ -860,8 +859,7 @@ func resourceUpdate(ctx context.Context, d *schema.ResourceData, meta any) diag.
 			if d.HasChange("config_server_management_mode") {
 				request.ConfigServerManagementMode = conversion.StringPtr(d.Get("config_server_management_mode").(string))
 			}
-			// can call latest API (2024-10-23 or newer) as autoscaling property is not specified, using older version just for caution until iss autoscaling epic is done
-			if _, _, err := connV220240805.ClustersApi.UpdateCluster(ctx, projectID, clusterName, request).Execute(); err != nil {
+			if _, _, err := connV2.ClustersApi.UpdateCluster(ctx, projectID, clusterName, request).Execute(); err != nil {
```

Comment:
can we preserve a comment here to warn that this update cannot contain auto scaling property? Was think something like this in my branch
`// can call latest API (2024-10-23 or newer) as replications specs with autoscaling property is not specified`

Reply:
Sounds good, added comments